torch/cuda/_memory_viz.py

```diff
@@ -89,7 +89,9 @@ def _block_extra(b):
 
 def format_flamegraph(flamegraph_lines, flamegraph_script=None):
     if flamegraph_script is None:
-        flamegraph_script = f"/tmp/{os.getuid()}_flamegraph.pl"
+        cache_dir = os.path.expanduser("~/.cache/")
```

Comment:
```suggestion
        cache_dir = os.getenv("XDG_CACHE_HOME", os.path.expanduser("~/.cache"))
```

Reply:
I though about it. Do you know when home would not be sufficient?